models/user/user.go

```diff
@@ -622,6 +596,7 @@ var (
 		"swagger.v1.json",
 		"user",
 		"v2",
+		"gitea-actions",
```

Comment:
What happens if this user already exists? Is there a need to add a migration? If so, what will the migration do? Assign a random username? Should possibly restrict all `gitea-*` usernames to avoid this in the future (say we add depandabot)?

Reply:
So that is why this is a breaking PR, I explained that in the [description](https://github.com/go-gitea/gitea/pull/21937#issue-1464325346).

I don't think there's much chance of that happening, and if it really happens, it's not too much trouble to rename the user.

> Should possibly restrict all gitea-* usernames to avoid this in the future?

I'm not sure, I think there will be a lot of opposition. Maybe we will do that in the future.

Reply:
Ok